browser/src/__tests__/NodeAutosuggest.test.tsx

```diff
@@ -1,22 +1,49 @@
-import {shallow, configure} from 'enzyme';
+import { configure} from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16'
 import {expect} from 'chai';
 import NodeAutoSuggestService from "../service/NodeAutosuggestService"
-import React from 'react';
 
 
 
 configure({adapter: new Adapter()});
 
 describe("Map of class to Provider has correct short name", () => {
-    test("should display a blank login form, with remember me checked by default", async () => {
-      // ???
-    });
 
-    test('sum 1 + 2 = 3' , () => {
-      const nodeContainer = shallow(<NodeAutoSuggestService/>);
-      expect((nodeContainer.instance() as NodeAutoSuggestService).add(1,2)).equals(3)
+    test('test 1', () =>{
+      const nodeAutoSuggest = new NodeAutoSuggestService();
+      expect(nodeAutoSuggest.parseString(
+        "java.util.Map<java.lang.Class<? extends com.snap.durablejob.DurableJob<?>>,javax.inject.Provider<"+
+        "com.snap.durablejob.DurableJobProcessor<?,?>>>")).equals(
+        "Map<Class<DurableJob<?>> Provider<DurableJobProcessor<?>>>"
+        )
     })
 
+    test('test 2', () => {
+      const nodeAutoSuggest = new NodeAutoSuggestService();
+      expect(nodeAutoSuggest.parseString("com.snap.dagger.browser.sample.CoffeeApp.CoffeeShop")).equals(
+        "CoffeeApp.CoffeeShop")
+    })
+
+    test('test 3', () => {
+      const nodeAutoSuggest = new NodeAutoSuggestService();
+      expect(nodeAutoSuggest.parseString(
+        "java.util.Map<java.lang.Class<? extends com.snap.durablejob.DurableJob<?>>,javax.inject.Provider<"+
```

Comment:
Is this input an actual key from ComponentManifests.json?

Reply:
https://ci-log-proxy.gae.sc-corp.net/log-proxy/snapengine-builder-artifacts/android-dagger-browser/latest/index.html#/com.snap.mushroom.dagger.durablejob.MushroomFeatureGraphDurableJobRegistry/node/java.util.Map%3Cjava.lang.Class%3C%3F%20extends%20com.snap.durablejob.DurableJob%3C%3F%3E%3E%2Cjavax.inject.Provider%3Ccom.snap.durablejob.DurableJobProcessor%3C%3F%2C%3F%3E%3E%3E

I got this dependency from this link you sent sometime ago